interp/runt.toml

```diff
@@ -3,19 +3,15 @@ ver = "0.3.2"
 # Check basic functionality of the interpreter
 [[tests]]
 name = "unit"
-paths = [
-  "tests/unit/*.futil"
-]
+paths = ["tests/unit/*.futil"]
```

Comment:
Any reason to keep the unit tests around if they have overlap with the correctness tests anyways?

Reply:
The unit tests don't really overlap with the correctness suite, unless I'm looking in the wrong places. Most of them are just basic sanity checks.